core/templates/dev/head/pages/profile-page/profile-page.directive.html

```diff
@@ -1,39 +1,41 @@
 <background-banner></background-banner>
 <div class="oppia-profile-container">
   <md-card class="oppia-profile-user-card">
-    <div class="oppia-profile-picture-container">
-      <div ng-if="$ctrl.profileIsOfCurrentUser" class="oppia-profile-picture">
-        <a href="/preferences" class="oppia-profile-preferences">
-          <div class="oppia-profile-picture-mask">
-            <div class="oppia-profile-picture-edit-prompt">
-              Edit avatar
-              <i class="material-icons md-18" title="Edit Profile Picture">&#xE254;</i>
+    <div class="oppia-profile-user-container">
```

Comment:
Can you please explain the changes in this file? Thanks!

Reply:
Hi, i'ved divided the elements inside the container into 2, and added a sperate container of the user content